plugins/source/aws/resources/services/identitystore/users.go

```diff
@@ -15,13 +18,23 @@ func Users() *schema.Table {
 	return &schema.Table{
 		Name:        tableName,
 		Description: `https://docs.aws.amazon.com/singlesignon/latest/IdentityStoreAPIReference/API_User.html`,
-		Resolver:    fetchIdentitystoreUsers,
+		Resolver:    fetchUsers,
 		Transform:   transformers.TransformWithStruct(&types.User{}),
 		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "identitystore"),
+		Columns: schema.ColumnList{
+			client.DefaultAccountIDColumn(true),
+			client.DefaultRegionColumn(true),
```

Comment:
I think we might want to change the `account_id` & `region` columns to their `request_` selves in a follow-up PR (it will definitely break some queries, though)

Reply:
a11ec67844f00f56bfbac40d39adf3657cb6115e